XiEditor/CoreConnection.swift

```diff
@@ -70,7 +70,15 @@ struct FileWriter {
     }
 }
 
-class CoreConnection {
+/**
+ Protocol used to properly test classes which require `CoreConnection` dependency.
```

Comment:
I might just say,

'Protocol describing the general interface with core. Concrete implementations  may be provided for different transport mechanisms, e.g. stdin/stdout, unix sockets, or FFI."